src/model/transformations/response.spec.jsx

```diff
@@ -135,7 +135,7 @@ describe('response tranformations', () => {
     expect(result.Datatype.Maximum).toEqual(maximum);
   });
   test('when Decimals is defined', () => {
-    const typeName = 'DATE';
```

Comment:
Why is it not `'DATE'` anymore?

Reply:
it actually does not change anything because we manage it in the same way whatever the type